lib/src/create_report.dart

```diff
@@ -778,46 +778,67 @@ Future<ReportSection> _multiPlatform(String packageDir, Pubspec pubspec) async {
       }).join(', ');
     }
 
+    _Subsection scorePlatforms(
+        Map<String, String> tagNames,
+        _Status Function(int) statusFromCount,
+        List<String> tags,
+        List<Explanation> explanations) {
+      final unofficialExplanations = explanations
+          .where((e) => e.tag != null && !tagNames.containsKey(e.tag));
+      final officialExplanations = explanations
+          .where((e) => e.tag == null || tagNames.containsKey(e.tag));
+      final paragraphs = [
+        if (officialExplanations.isNotEmpty)
+          _RawParagraph(
+              'Consider supporting multiple platforms. Points were deducted due to the following issues:\n'),
```

Comment:
Perhaps "issues" is a bit harsh? Maybe just omit it, so "Points were deducted due to the following:" ?

Reply:
Done